script/build-windows.ps1

```diff
@@ -29,7 +29,6 @@
 #        .\script\build-windows.ps1
 
 $ErrorActionPreference = "Stop"
-Set-PSDebug -trace 1
```

Comment:
This is helpful piece of context, can you add it to the commit message so we don't lose it when we merge please?